src/components/Stats/Volume/Volume.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react'
+import { Grid, Typography, Box } from '@material-ui/core'
+import { formatNumbers, showPrefix } from '@consts/utils'
+import { ResponsiveBar } from '@nivo/bar'
+import classNames from 'classnames'
+import { useStyles } from './style'
+import { colors } from '@static/theme'
+
+interface StatsInterface {
+  percentVolume: number
+  volume: number
+  data: Array<{
+    timeStamp: number
+    value: number
+  }>
+}
+
+const Volume: React.FC<StatsInterface> = ({ percentVolume, volume, data }) => {
+  const classes = useStyles()
+
+  const cutArray = (arr: Array<{ timeStamp: string; value: number[] }>, size: number) => {
+    const arrData = arr.slice(0)
+    const result = []
+
+    while (arrData.length > 0) result.push(arrData.splice(0, size))
+
+    return result
+  }
+
+  function concatArray<T>(arr: T[][]) {
+    return arr.reduce(
+      (array, isArray) => (Array.isArray(isArray) ? array.concat(isArray) : array),
+      []
+    )
+  }
+
+  const converToUnix = data
+    .map(el => {
+      const unix = el.timeStamp
+      const date = new Date(unix)
+      const hours = date.getHours()
+
+      const convertedHour = hours >= 13 ? `${hours - 12}PM` : `${hours}AM`
+
+      return { ...el, timeStamp: convertedHour }
+    })
+    .filter(({ timeStamp }) => timeStamp !== '0AM')
```

Comment:
instead of removing such elements, convert timestamp to 12PM

Reply:
done